index.html

```diff
@@ -70,11 +70,11 @@
 				<svg class="triangle" viewBox="0 0 100 100" preserveAspectRatio="none" aria-hidden="true"><path d="M0 0h100l-50 100"/></svg>
 				<!-- Start slide -->
 				<div class="da-slide da-slide-current">
-					<h2>DietPi v7 has been released!</h2>
+					<h2>DietPi 7 is now released !</h2>
 					<h4>Essential upgrade to be in sync with latest platform updates</h4>
-					<p>Try out this latest version! More changes together with new software titles: Steam is now available for ARM boards, together with a lightweight MJPEG streamer. You can also run Docker Compose on DietPi to run multi-container Docker applications.</p>
-					<a href="https://dietpi.com/docs/release-notes/#february-2021-version-70" class="da-link" target="_blank" rel="noopener">Learn more</a>
-					<div class="da-img"><img class="img-fluid" src="images/dietpi-logo_360x360.png" alt="DietPi logo" width="300" height="300" loading="lazy"></div>
+					<p>DietPi brings today more joy enabling Steam & Box86, and it also makes your SBC more powerful, running multi-container apps with Docker Compose ! Checkout the latest DietPi updates now !</p>
+					<a href="https://dietpi.com/docs/release-notes/#february-2021-version-70" class="da-link" target="_blank" rel="noopener">See what's new</a>
+					<div class="da-img"><img class="img-fluid" src="images/dietpi-logo_360x360.png" width="300" alt="DietPi logo" loading="lazy"></div>
```

Comment:
Please keep the explicit image height here (like it is now everywhere else) to avoid content shift on lazy or failing image loading.

Actually I liked the text how it was 😄:
- "enabling Steam & Box86" is a bit imprecise, since Steam was available before. "Steam is now available for ARM boards" (via Box86) is clearer.
- "and it also makes your SBC more powerful, running multi-container apps" sounds a bit like it was a general enhancement for all SBCs, but Docker Compose "only" enables one to run a specific type of container apps.

What about:
```
DietPi v7 brings more joy enabling Steam on ARM via Box86, and Docker Compose to manage multi-container apps. As well a lightweight MJPEG camera streamer has been added. Checkout the latest DietPi update now!
```
?

Reply:
OK. Let's review it tomorrow